scene/gui/text_edit.cpp

```diff
@@ -2421,7 +2422,7 @@ String TextEdit::get_tooltip(const Point2 &p_pos) const {
 }
 
 void TextEdit::set_tooltip_request_func(Object *p_obj, const StringName &p_function, const Variant &p_udata) {
-	tooltip_obj = p_obj;
+	tooltip_obj_id = p_obj->get_instance_id();
```

Comment:
Should this check for `p_obj`'s validity first?

Reply:
Ah, yes 🤣